XenAdmin/Wizards/NewVMWizard/Page_CpuMem.cs

```diff
@@ -44,7 +44,15 @@ namespace XenAdmin.Wizards.NewVMWizard
     public partial class Page_CpuMem : XenTabPage
     {
         private VM Template;
-        int memoryMode = 1;  // number of spinners to show: 1 for just "memory", 2 for "minimum" and "maximum", 3 for "static max" too
+
+        // number of spinners to show
+        enum MemoryMode {

```

Comment:
C# style: we prefer brackets on a separate line and spaces between operators a values assigned.